plugins/org.eclipse.osee.disposition.rest.model/src/org/eclipse/osee/disposition/model/DispoAnnotationData.java

```diff
@@ -121,22 +121,37 @@ public boolean getIsConnected() {
    }
 
    public String getResolution() {
+      if (resolution == null) {
```

Comment:
you can add a constructor that accepts/sets all of these so that you don't have to set some to ""/false at object variable declaration above (line 45 e.g.). Then you can avoid all these null checks. Should do null checking within the constructor to make sure everything is set.